client/my-sites/customer-home/cards/actions/quick-links-for-ecommerce-sites/index.jsx

```diff
@@ -0,0 +1,194 @@
+import { getAllFeaturesForPlan } from '@automattic/calypso-products/';
+import { JetpackLogo, FoldableCard } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useEffect } from 'react';
+import { useDispatch, useSelector, connect } from 'react-redux';
+import { useDebouncedCallback } from 'use-debounce';
+import { savePreference } from 'calypso/state/preferences/actions';
+import { getPreference } from 'calypso/state/preferences/selectors';
+import { canCurrentUser } from 'calypso/state/selectors/can-current-user';
+import getSelectedEditor from 'calypso/state/selectors/get-selected-editor';
+import isSiteAtomic from 'calypso/state/selectors/is-site-wpcom-atomic';
+import isSiteWpcomStaging from 'calypso/state/selectors/is-site-wpcom-staging';
+import { getSitePlanSlug, getSite, getSiteOption } from 'calypso/state/sites/selectors';
+import getSiteAdminUrl from 'calypso/state/sites/selectors/get-site-admin-url';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import { trackAddDomainAction, trackManageAllDomainsAction } from '../quick-links';
+import ActionBox from '../quick-links/action-box';
+import '../quick-links/style.scss';
+
+const QuickLinksForHostedSites = ( props ) => {
```

Comment:
It's missing a rename from QuickLinksForHostedSites to QuickLinksForEcommerceSites.
```suggestion
const QuickLinksForEcommerceSites = ( props ) => {
```

Reply:
Thanks!